crates/re_data_ui/src/instance_path.rs

```diff
@@ -27,52 +24,88 @@ impl DataUi for InstancePath {
         else {
             if ctx.recording().is_known_entity(entity_path) {
                 // This is fine - e.g. we're looking at `/world` and the user has only logged to `/world/car`.
-                ui.label(format!(
-                    "No components logged on timeline {:?}",
-                    query.timeline().name()
-                ));
+                label_for_ui_layout(
+                    ui,
+                    ui_layout,
+                    format!(
```

Comment:
could this be a member method?

```suggestion
                ui_layout.label(
                    ui,
                    format!(
```

Reply:
Actually, both this one and `table_for_ui_layout` could be methods. I like that.